db.go

```diff
@@ -68,7 +67,7 @@ type DB struct {
 	valueDirGuard *directoryLockGuard
 
 	closers   closers
-	elog      trace.EventLog
+	logger    Logger
```

Comment:
You can remove this. The `options` struct has the `logger`. Look at `db.opt.Logger`.